client/src/components/base/Toast.tsx

```diff
@@ -7,13 +7,23 @@ export default function Toast({
   children,
   show,
   error = false,
+  fadeOut = false,
   onClose,
 }: {
   children: JSX.Element;
   show: boolean;
   error?: boolean;
+  fadeOut?: boolean;
   onClose: () => void;
 }) {
+  useEffect(() => {
+    if (fadeOut) {
+      setTimeout(() => {
+        onClose();
+      }, 3000);
+    }
```

Comment:
@schultztimothy I think we can return a function in `useEffect` that is executed when the component is unmounted, so that we can cancel the timeout. otherwise if it's re-rendered we will have multiple timeout. `setTimeout` returns an id, so we could do something like:

```
return () => clearTimeout(timeoutID)
```